routes/organizations.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const config = require('../config/server');
+const SCHEMAS = require('../lib/schemas');
+const Organizations = require('../handlers/organizations');
+
+const routes = [];
+const API_BASE_PATH = `${config.apiPrefix}/organizations`;
+
+// GET /organizations
+routes.push({
+  method: 'GET',
+  path: API_BASE_PATH,
+  config: {
+    handler: Organizations.listOrganizations,
+    description: 'List all organizations',
+    notes: 'List all organizations in the system',
+    plugins: {
+      'hapi-swagger': {
+        responses: {
+          '200': {
+            description: 'Success',
+            schema: SCHEMAS.Organizations
+          },
+          '400': {
```

Comment:
There's no scenario in which a 400 can happen. However, you are missing the 401, the 403, and the 500. You can see this as an example: https://github.com/adoptapanama/admin-api/blob/develop/routes/users.js#L29-L39